src/20250120_Multi_FebruaryFundingUpdate/FebruaryFundingUpdate.md

```diff
@@ -0,0 +1,146 @@
+
+Finally, as part of the revamping of SKY, certain DAI-correlated assets will be swapped into aEthUSDS.
+
+## Specification
+
+### Transfer FLUID
+
+Transfer an initial 25% of the 1/3 FLUID holdings assigned to support GHO's growth and adoption on Fluid.
+
+Specifically, Transfer 95,417 $FLUID to Merit SAFE `0xdeadD8aB03075b7FBA81864202a2f59EE25B312b`
+
+Reference this prior [proposal](https://governance.aave.com/t/arfc-fluid-alignment-with-inst-purchase/19921) for more information.
+
+### Gas Reimbursement and EURA Bad Debt Clearing Reimbursement
+
+Transfer 26,500 USDC, 3.404 ETH and 149 PYUSD to ACI to reimburse expenses incurred.
+
+This reimburses associated expenses with clearing EURA bad debt, gas costs linked to AIP submissions and administering the PYUSD rewards program respectively.
+
+ACI Receiving Address: `0xac140648435d03f784879cd789130F22Ef588Fcd`
+
+### Migrate funds from Aave v2 to v3
+
+Migrate the following assets from Aave v2 to v3 on the respective network.
+
+Where it says All-1, it refers to all available liquidity minus 1 unit as to not drain the pool where possible. In other pools, it will be as much as is available to withdraw.
+
+|   Ethereum    |    Avalanche    |     Polygon     |
+| :-----------: | :-------------: | :-------------: |
+| aWETH (All-1) | avWETH (All-1)  | amWMATIC (All-1)|
+| aWBTC (All-1) |  avDAI (All-1)  | amUSDT (All-1)  |
+| aLINK (All-1) | avWAVAX (All-1) |                 |
+| aMKR (All-1)  |                 |                 |
+| aUSDC (1.5M)  |                 |                 |
+| aUSDT (2.0M)  |                 |                 |
+
+Funds are to be deposited into the Core deployment on Ethereum and Aave v3 on the respective networks.
+
+Note: Not all USDC and USDT on v2 Ethereum can be migrated due to high utilisation. This publication proposes a partial migration subject to liquidity availability.
+
+### Deposit Passively Held Assets into Aave
+
+Deposit the following assets into their respective V3 Pools.
+
+| Ethereum v3 (Core) | Optimism v3 | Avalanche v3 | Polygon v3 |
+| :----------------: | :---------: | :----------: | :--------: |
+|        USDC        |    USDC     |     USDC     |   wMATIC   |
+|        USDT        |    wETH     |     USDT     |    wBTC    |
+|                    |    USDT     |    wAVAX     |    USDT    |
+|                    |             |    wETH.e    |    LINK    |
+|                    |             |    BTC.d     |            |
+
+### Acquire ETH and deposit into Aave v3 (Core)
+
+Withdraw and swap the following assets to ETH.
+
+|     Ethereum     |
+| :--------------: |
+| aEthLUSD (All-1) |
+|   aDPI (All-1)   |
+
+### Acquire GHO and deposit into Aave v3 (Prime)
+
+Both withdrawn above from Aave V2 Ethereum.
+
+|   Ethereum   |
+| :----------: |
+| USDT (1.50M) |
+| USDC (1.50M) |
+
+### Acquire USDS and deposit into Aave V3 (Core)
+
+Withdraw and swap the following assets to USDS.
+
```

Comment:
same here. will this be done in later AIP?

Reply:
correct, this is why the md just mentions the swap. Will include a note that the deposits will happen at a later date